Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -1,15 +1,44 @@
 import SwiftUI
 import UIKit
 
-public struct SimplifiedLoginViewModel {
+public struct SimplifiedLoginUIFactory {
+    
+    public static func buildViewController(client: Client,
+                                           env: ClientConfiguration.Environment,
+                                           clientStartLoginSession: @escaping () -> Void ) -> UIViewController {
+        let viewModel = SimplifiedLoginViewModel(client: client, env: env)! // TODO: throw error
+        let s = SimplifiedLoginViewController(viewModel: viewModel )
+        
+        viewModel.onClickedContinueAsUser = {} // TODO:
+        viewModel.onClickedSwitchAccount = { clientStartLoginSession() }
+        viewModel.onClickedContinueWithoutLogin = { s.dismiss(animated: true, completion: nil) }
+        viewModel.onClickedPrivacyPolicy = {}  // TODO:
+        
+        return s
+    }
+}
+
+
+class SimplifiedLoginViewModel {
+    
+    var onClickedContinueWithoutLogin: (() -> Void)?
+    var onClickedSwitchAccount: (() -> Void)?
+    var onClickedPrivacyPolicy: (() -> Void)?
+    var onClickedContinueAsUser: (() -> Void)? // TODO:
+    
     var localisation: Localisation
     var iconNames: [String]
     let schibstedLogoName = "sch-logo"
     
     let displayName = "Daniel.User" // TODO: Need to be fetched
     let clientName = "Finn" // TODO: Need to be fetched
     
-    init?(env: ClientConfiguration.Environment){
+    let client: Client
+    
+    init?(client: Client, env: ClientConfiguration.Environment) {
```

Comment:
client has already configuration inside. Does it differ here?

Reply:
I will change this. I am currently thinking if the Client should be injected.